ravenframework/Distributions.py

```diff
@@ -488,11 +488,8 @@ def rvs(self, size=None):
       @ In, size, int, optional, number of entries to return (one if None)
       @ Out, rvsValue, float or list, requested random number or numbers
     """
-    if size is None:
-      rvsValue = self.ppf(random())
-    else:
-      # TODO to speed up, do this on the C side instead of in python
-      rvsValue = np.array([self.rvs() for _ in range(size)])
+    size = 1 or size
```

Comment:
This is equivalent to: `size = 1` since 1 is always true in python, and so `1 or size` short circuits to `1`

Reply:
Thanks for catching that. Inverting that to `size or 1` is what was intended.